crates/pop-parachains/src/new_parachain.rs

```diff
@@ -75,12 +80,72 @@ pub fn instantiate_standard_template(
 	Ok(tag)
 }
 
+/// Create a new tanssi compatible container chain.
+///
+/// # Arguments
+///
+/// * `template` - template to generate the parachain from.
+/// * `target` - location where the parachain will be created.
+/// * `tag_version` - version to use (`None` to use latest).
+fn instantiate_tanssi_template(
+	template: &Parachain,
+	target: &Path,
+	tag_version: Option<String>,
+) -> Result<Option<String>> {
+	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
+	let source = temp_dir.path();
+	let tag = Git::clone_and_degit(template.repository_url()?, &source, tag_version)?;
+
+	// Relevant files to extract.
+	let files =
+		[".rustfmt.toml", "Cargo.toml", "Cargo.lock", "LICENSE", "README.md", "rust-toolchain"];
```

Comment:
Feels like this should be defined on the template via enum properties ultimately, once a stable solution is identified.

Reply:
Though, as it stands right now the "only" option would be using a single string that includes all these files under in one concrete property. Scrum docs advise against using a big number of properties, and support for types that are not strings is experimental. :/